apps/web/pages/settings/my-account/out-of-office/index.tsx

```diff
@@ -0,0 +1,367 @@
+import { DateRangePicker } from "@tremor/react";
+import React, { useState } from "react";
+import { useForm, useFormState } from "react-hook-form";
+
+import dayjs from "@calcom/dayjs";
+import SectionBottomActions from "@calcom/features/settings/SectionBottomActions";
+import { getLayout } from "@calcom/features/settings/layouts/SettingsLayout";
+import { ShellMain } from "@calcom/features/shell/Shell";
+import { UpgradeTip } from "@calcom/features/tips";
+import { CAL_URL } from "@calcom/lib/constants";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { trpc } from "@calcom/trpc/react";
+import useMeQuery from "@calcom/trpc/react/hooks/useMeQuery";
+import { Button, ButtonGroup, Meta, showToast, Badge, Select, SkeletonText } from "@calcom/ui";
+import { TableNew, TableBody, TableCell, TableHead, TableHeader, TableRow } from "@calcom/ui";
+import { FastForward, Link, Moon, Send, Trash2 } from "@calcom/ui/components/icon";
+
+import PageWrapper from "@components/PageWrapper";
+
+import "./DateSelect.css";
```

Comment:
this breaks the page for me: 

<img width="1160" alt="Screenshot 2023-12-18 at 4 24 06 PM" src="https://github.com/calcom/cal.com/assets/30310907/c75a6eb2-d2f4-4a13-a5dd-9399c4fabdf0">

Reply:
I think for this I will go for the .module.css route.
https://nextjs.org/docs/pages/building-your-application/styling/css-modules#example